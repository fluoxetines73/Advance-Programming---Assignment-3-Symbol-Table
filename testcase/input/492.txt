BEGIN
RPRINT
BEGIN
INSERT g_1 string
LOOKUP g_1
PRINT
LOOKUP g_1
ASSIGN g_1 '1234ABC'
PRINT
RPRINT
INSERT f_FF string
BEGIN
BEGIN
PRINT
RPRINT
LOOKUP f_FF
BEGIN
END
BEGIN
ASSIGN g_1 'abcABC'
PRINT
LOOKUP f_FF
PRINT
LOOKUP f_FF random
RPRINT
END
END
END
END
END