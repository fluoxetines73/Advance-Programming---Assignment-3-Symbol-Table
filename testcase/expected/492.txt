Invalid: LOOKUP f_FF random